src/utils/composeEventHandlers.js

```diff
@@ -1,5 +1,5 @@
-const composeHandler = (...args) => e => {
-  args.forEach(fn => fn && fn(e));
+const composeHandler = (...fns) => (...args) => {
```

Comment:
This function is intended to invoke multiple event handlers with a singular event object, rather than invoking multiple functions with any arguments. Can you please revert this change?

Reply:
Done